runtime/Cargo.toml

```diff
@@ -51,6 +51,10 @@ pallet-contracts = { git = "https://github.com/paritytech/substrate", package =
 pallet-contracts-primitives = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-primitives", default-features = false }
 pallet-contracts-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc-runtime-api", default-features = false }
 
+# Chain extension
+pallet-assets = { git = "https://github.com/paritytech/substrate", package = "pallet-assets", default-features = false }
```

Comment:
I think you forgot to propagate the `runtime-benchmarks` feature.

Reply:
I used the wrong hash commit=)

Reply:
> I think you forgot to propagate the `runtime-benchmarks` feature
👆